packages/react-variant-view/src/util/VariantUtil.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    getProteinPositionFromProteinChange,
+    Mutation,
+} from 'cbioportal-utils';
+import { VariantAnnotationSummary } from 'genome-nexus-ts-api-client';
+import { getTranscriptConsequenceSummary } from './AnnotationSummaryUtil';
+
+export function variantToMutation(
```

Comment:
We may want to move some of these utility functions into `cbioportal-utils`, but I guess it is okay to leave them here at this point.